vars/deployCDK.groovy

```diff
@@ -27,7 +34,11 @@ def call(Map conf) {
             sh "juju add-model -c ${conf.controller} ${conf.model}"
         }
     }
-    sh "juju deploy -m ${conf.controller}:${conf.model} ${conf.bundle} --overlay ${conf.version_overlay} --channel ${conf.bundle_channel}"
+    // There is no way to tell Juju to use a specific channel of a bundle but
+    // to leave the contents unmodified, so we have to pull a local copy of the
+    // channel that we want.
+    sh "charm pull ${conf.bundle} --channel ${conf.bundle_channel} ./bundle-to-test"
```

Comment:
There could be a problem here if `/bundle-to-test` already exists, but I'm not sure if that'll be a problem in practice -- maybe the workspace is clean when `deployCDK` is called? Dunno.